src/Illuminate/Routing/RouteCollection.php

```diff
@@ -4,6 +4,7 @@
 
 use ArrayIterator;
 use Countable;
+use Illuminate\Container\Container;
```

Comment:
Shouldn't this reference the container contract (along with all the docblocks as well)?

Reply:
Yeah true thanks. I'm refactoring a lot here btw but will take that into account.

Reply:
👍

Reply:
Just checked and I unfortunately need to pass the actual container here because the Route atm requires this. I'm thinking of refactoring that as well but that'll be for a different pr.